### PR TITLE
docs: add the CHANGELOG for v2.12.3

### PR DESCRIPTION
**Pull request by tao12345666333**
**What this PR does / why we need it**:

**Which issue this PR fixes**:

**Special notes for your reviewer**:

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [x] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -381,6 +382,21 @@ Adding a new version? You'll need three changes:
 [KongIngress to KongUpstreamPolicy migration guide]: https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/migrate/kongingress/
 [Migrate Credential Type Labels]: https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/migrate/credential-kongcredtype-label/
 
+## [2.12.3]
+
+> Release date: 2023-12-19
+
+### Fixed
+
+- Fix(manager): set InitCacheSyncDuration to 5s by default and allow it to be configured via cli.
```
Comment (pmalek) [inline CHANGELOG.md:391]: Can we add the CLI flag name here as it should be used on the command line?

Comment (tao12345666333) [inline CHANGELOG.md:391]:
```suggestion
- Fix(manager): set InitCacheSyncDuration to 5s by default and allow it to be configured via `-init-cache-sync-duration` CLI flag.
```

Is this okay?

Comment (tao12345666333) [reply CHANGELOG.md:391]:
```suggestion
- Fix(manager): set InitCacheSyncDuration to 5s by default and allow it to be configured via `-init-cache-sync-duration` CLI flag.
```

Is this ok?

Comment (pmalek) [reply CHANGELOG.md:391]:
```suggestion
- Fix(manager): set InitCacheSyncDuration to 5s by default and allow it to be configured via `--init-cache-sync-duration` CLI flag
```

Using double dash, just like it's shown when calling KIC's binary with `-h` or `--help`

**Review by randmonkey — APPROVED** [review]

**Pull request merged**